docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+
+### Installing menus
+
+### Making menus accessible
+
+Flutter's list component APIs support labeling for accessibility. For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Anatomy
+
+The following are menu anatomy diagrams showing all possible elements:
+
+![Menus anatomy diagrams](assets/menus/menus-anatomy-composite.png)
+
+1. Container
+1. Leading icon (optional)
+1. Text
+1. Divider (optional)
+1. Command (optional)
+1. Cascading menu indicator (optional)
+1. Selection state (optional)
+
+### Container attributes
+
+&nbsp; | Property
+------ | ---------
+**Color** | Use the `PopupMenuButton` widget. Within this widget use the `Colors` property
+**Height** | Use the `PopupMenuButton` widget. Within this widget use the `padding` property.
+**Overlay window** | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder` property. Each item added to this list will add an additional window.
+
+### Leading icon attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding Icon | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `leading` parameter to an `Icon`.
+
+### Text attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `title` parameter to a `Text` widget.
+**Color**      | Follow the steps above and then within the Text widget fill out the `style` parameter.
+**Typography** | Follow the steps above and then within the Text widget fill out the `style` parameter.
+
+### Divider attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Add Divider | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuDivider`.
+
+### Command attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding Icon | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `trailing` parameter to an `Icon`.
+
+### Cascading menu indicator attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding cascading menu | Not currently supported.
+
+### Selection state attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+ | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `CheckedPopupMenuItem`.
```

Comment:
Row here is missing the value of the first column, Or should it be empty? Doesn't look correct when I preview.

Reply:
Done